COLLABORATOR_GUIDE.md

```diff
@@ -411,64 +411,45 @@ recommended but not required.
-  notice indicating the deprecated status is added to the API documentation
-  but no functional changes are implemented in the code. There will be no
-  runtime deprecation warnings emitted for such deprecations by default.
-  Documentation-only deprecations may trigger a runtime warning when Node.js
-  is started with the [`--pending-deprecation`][] flag or the
-  `NODE_PENDING_DEPRECATION=1` environment variable is set.
-
-* *Runtime Deprecation* refers to the use of process warnings emitted at
-  runtime the first time that a deprecated API is used. A command-line
-  switch can be used to escalate such warnings into runtime errors that will
-  cause the Node.js process to exit. As with Documentation-Only Deprecation,
-  the documentation for the API must be updated to clearly indicate the
-  deprecated status.
-
-* *End-of-life* refers to APIs that have gone through Runtime Deprecation and
-  are no longer subject to the semantic versioning rules used by the project.
-  Backward-incompatible changes including complete removal of such APIs may
-  occur at any time.
-
-Documentation-Only Deprecations may be handled as semver-minor or semver-major
-changes. Such deprecations have no impact on the successful operation of running
-code and therefore should not be viewed as breaking changes.
-
-Runtime Deprecations and End-of-life APIs (internal or public) must be
-handled as semver-major changes unless there is TSC consensus to land the
-deprecation as a semver-minor.
-
-All Documentation-Only and Runtime deprecations will be assigned a unique
-identifier that can be used to persistently refer to the deprecation in
-documentation, emitted process warnings, or errors thrown. Documentation for
-these identifiers will be included in the Node.js API documentation and will
-be immutable once assigned. Even if End-of-Life code is removed from Node.js,
-the documentation for the assigned deprecation identifier must remain in the
-Node.js API documentation.
+* *Documentation-Only Deprecation*: A deprecation notice is added to the API
+  documentation but no functional changes are implemented in the code. There
+  will be no runtime deprecation warnings emitted for such deprecations by
+  default. (They may trigger a runtime warning when used with the
+  [`--pending-deprecation`][] CLI option or the `NODE_PENDING_DEPRECATION`
+  environment variable.)
+
+* *Runtime Deprecation*: A warning is emitted at runtime the first time that the
+  deprecated API is used. The [`--throw-deprecation`][] CLI option can be used
+  to treat these warnings as runtime errors. Documentation for the API must
+  indicate that the deprecation is a Runtime Deprecation.
+
+* *End-of-life*: The API is no longer subject to semantic versioning rules.
+  Backward-incompatible changes (including removal of the API) may occur at any
+  time. Documentation for the API must indicate that the API is End-of-life.
```

Comment:
When being on this: we might want to reword this that the backward-incompatible change should be done when moving something to EOL.